ci/release_notes.md

```diff
@@ -1,10 +1,107 @@
-## Bug Fixes
-* CRITICAL: A distributed Minio manifest would only deploy 1 Minio instance.
-* Fixed a typo in the post-deploy
 
-## New Features
-* `s3` addon command, which will automatically set your envvars and run commands
-  against `s3`.
+# Overview
+This release updates CF components to more recent versions. Most
+notably, this version brings the following related CF changes:
+* Diego now uses mTLS for communication.
+* CAPI now supports IAM Service Accounts when authenticating with
+  Google Cloud Storage
+* Scalable Syslog is now used, allowing multiple VMs to handle log
+  output.
 
-## Changes
-* The default port for Minio is now `443` instead of `9000`
\ No newline at end of file
+
+To accommodate these changes, the following kit-related changes were
+made:
+* 3 new databases are necessary: `silkdb`, `locketdb`, and 
+  `policyserverdb`
+* Parameters for `syslogger` VM type and instance count
+* Self-signed certs for internal mTLS communication 
```

Comment:
These are hands-off (I believe) and do not require operator involvement, so they don't need to be in the release notes.

Related: do we need to mention the SAN changes and how those will be handled?